app/code/core/Mage/Adminhtml/Block/Sales/Order/Grid.php

```diff
@@ -141,11 +141,15 @@ protected function _prepareColumns()
             );
         }
 
-        if (Mage::helper('catalog')->isModuleEnabled('Mage_Rss') &&
-            Mage::helper('rss')->isRssEnabled() &&
-            Mage::getStoreConfigFlag('rss/order/new')
-        ) {
-            $this->addRssList('rss/order/new', Mage::helper('sales')->__('New Order RSS'));
+        if (Mage::helper('sales')->isModuleOutputEnabled('Mage_Rss')) {
+            $filterString = $this->getParam($this->getVarNameFilter());
+            $filter = Mage::helper('adminhtml')->prepareFilterString($filterString);
+            $storeId = array_key_exists('store_id', $filter) ? $filter['store_id'] : null;
+
+            if (Mage::helper('rss')->isRssAdminOrderNewEnabled($storeId)) {
```

Comment:
if ($storeId && Mage::helper('rss')->isRssAdminOrderNewEnabled($storeId)) {

Reply:
$storeId  is used one line later.

```
            if (Mage::helper('rss')->isRssAdminOrderNewEnabled($storeId)) {
                $slug = $storeId ? '/store/' . $storeId : '';
                $this->addRssList('rss/order/new' . $slug, Mage::helper('sales')->__('New Order RSS'));
            }
```